capnpy/__init__.py

```diff
@@ -1,14 +1,16 @@
 import sys
 import pkg_resources
-from capnpy.compiler.compiler import DynamicCompiler
```

Comment:
if there is no compelling reason, I think it's better to keep all the imports together, before running the code to get the version